openedx/features/enterprise_support/api.py

```diff
@@ -233,52 +259,87 @@ def enterprise_enabled():
     return 'enterprise' in settings.INSTALLED_APPS and getattr(settings, 'ENABLE_ENTERPRISE_INTEGRATION', True)
 
 
-def enterprise_customer_for_request(request, tpa_hint=None):
+def enterprise_customer_for_request(request):
     """
     Check all the context clues of the request to determine if
     the request being made is tied to a particular EnterpriseCustomer.
     """
+
     if not enterprise_enabled():
         return None
 
     ec = None
+    sso_slug = request.GET.get('tpa_hint')
 
     running_pipeline = get_partial_pipeline(request)
     if running_pipeline:
         # Determine if the user is in the middle of a third-party auth pipeline,
-        # and set the tpa_hint parameter to match if so.
-        tpa_hint = Registry.get_from_pipeline(running_pipeline).provider_id
+        # and set the sso_slug parameter to match if so.
```

Comment:
i'm curious why the name of this has changed. not opposed, but i don't want to introduce more inconsistencies and i also want to be sure that this is the name we will call it in future discussions/changes.